ui/shared/components/panel/variants/selectors.js

```diff
@@ -82,13 +85,23 @@ export const getPairedSelectedSavedVariants = createSelector(
       )].filter(varGuid => selectedVariantsByGuid[varGuid])
 
       if (variantCompoundHetGuids.length) {
-        const unseenGuids = variantCompoundHetGuids.filter(varGuid => !seenCompoundHets.includes(varGuid))
-        const compHet = [variant, ...unseenGuids.map(varGuid => selectedVariantsByGuid[varGuid])].sort(
-          // sorts manual variants to top of list, as manual variants are missing all populations
-          (a, b) => (a.populations ? 1 : 0) - (b.populations ? 1 : 0),
-        )
-        acc.push(compHet)
+        let unseenGuids = variantCompoundHetGuids.filter(varGuid => !seenCompoundHets.includes(varGuid))
```

Comment:
`unseenGuids` => `unseenCompoundHetGuids`. Current variable name unclear about what these guids are.

Reply:
but they are not necessarily comp hets, they might be MNVs or complex SVs which also have multiple variants, and we are being really inconsistant about that in the variable names in this functions. I'll update all the variables using `CompoundHet` to `Paired` in this block